pyvista/plotting/plotting.py

```diff
@@ -618,6 +621,10 @@ def render(self):
         """
         if hasattr(self, 'ren_win') and not self._first_time:
             self.ren_win.Render()
+        # Not sure if this is ever needed but here as a reminder
+        # if hasattr(self, 'iren') and not self._first_time:
+        #     self.iren.Render()
+        return
```

Comment:
I put that comment as we had some really old code that called `self.iren.Render()` and neither @akaszynski or I could remember why or even if it's ever needed. I moved that code here just in case we have issues down the road and to remember that we can call Render on the `iren` but should never do it outside of `BasePlotter.render`'s context

The empty return was just a personal code style thing... it doesn't change how the method returns, just breaks the code up a little nicer IMO